# 异常调试
需要的关键字：try\except\raise\finally

## 不仅显示提示，同时显示异常信息：
```python
try:
	print(1/'a')
except ZeroDivisionError as e:
	print（'0不能做除数 %s' %e）
```

## 捕获多种异常信息，将所有error放入元组中：
```python
except (ValueError,TypeError):
	print('0不能做除数')
```

## 捕获全部异常信息 Exception关键字：
```python
except Exception as e:
	print(e)
```

## 自定义错误信息 raise关键字：
```python
try：
	raise NameError('helloError')
except NameError:
	print('my custom error')
```

## 不论发生什么错误都要做的写入finally:
```python
try:
	a = open('name.txt')
except Exception as e:
	print(e)
finally:
	a.close()
```